PowerPoint_Adapter/AdapterActions/Push.cs

```diff
@@ -83,10 +82,9 @@ public override List<object> Push(IEnumerable<object> objects, string tag = "",
                 {
                     presentationDoc = PresentationDocument.Open(memoryStream, true);
                 }
-                catch (Exception e)
+                catch (Exception ex)
                 {
-                    memoryStream.Close();
-                    BH.Engine.Base.Compute.RecordError("Could not open the file: " + e.Message);
+                    BH.Engine.Base.Compute.RecordError(ex, "An error occurred while opening the presentation.");
```

Comment:
Can we use the word `template` instead of `presentation` to make it clearer to the user what file this error is related to ?
Can we also keep the exception's message in the error message? This can help understand where the error is coming from when in the UI.

Reply:
The `RecordError` method includes the exception message in the error message, as below. Though I will make the message itself clearer.
![image](https://github.com/user-attachments/assets/bd786191-28e4-4817-bdbe-2a812ee74f9a)

Reply:
Ah, right! I had forgotten about that feature.